src/transport/SessionManager.h

```diff
@@ -621,6 +621,10 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate, public FabricTabl
         return payloadHeader.HasMessageType(Protocols::SecureChannel::MsgType::MsgCounterSyncReq) ||
             payloadHeader.HasMessageType(Protocols::SecureChannel::MsgType::MsgCounterSyncRsp);
     }
+
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
+    void MarkSecureSessionDefunctForConnection(Transport::ActiveTCPConnectionState * conn, CHIP_ERROR conErr);
+#endif // INET_CONFIG_ENABLE_TCP_ENDPOINT
```

Comment:
nit: move this above IsControlMessage so that all non static member functions stay together